core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -505,13 +505,16 @@ public TableMetadata getImportTableMetadata(String namespace, String table)
     }
 
     try (Connection connection = dataSource.getConnection()) {
+      String rawCatalogName = rdbEngine.getCatalogName(namespace);
+      String rawSchemaName = rdbEngine.getSchemaName(namespace);
```

Comment:
We should change the local variable names as well, right?
```suggestion
      String catalogName = rdbEngine.getCatalogName(namespace);
      String schemaName = rdbEngine.getSchemaName(namespace);
```

Reply:
Thanks! Addressed it in https://github.com/scalar-labs/scalardb/pull/2001/commits/8f50dc6ec83f901f9d925ed3a22505eef09b44c3